resources/rules.k.tmpl

```diff
@@ -1,45 +1,35 @@
 // -*- mode: k3; -*-
 // VERIFICATION.k
 
-    rule #rangeUInt    (   1 ,      X ) => #range ( 0               <= X <= 1               ) [macro]
+    // Memory Abstraction
```

Comment:
Could be nice to make this a markdown file and extract the K stuff as part of the build process.

Reply:
In the end the change was so small I decided not to include the refactor, for the sake of clarity. Agree it could be a nice to have though.